openemory/harvest/entrez.py

```diff
@@ -95,7 +96,8 @@ def _query(self, base_url, qargs, response_xmlclass):
         # policies.
         qurl = base_url + urlencode(qargs)
         logger.debug('EntrezClient querying: ' + qurl)
-        return xmlmap.load_xmlobject_from_file(qurl,
+        target_file = urllib2.urlopen(qurl)
```

Comment:
Assuming that qurl is always a url, we should always open the file so that it could be read correctly with the xmlmap.load_xmlobject_from_file method. But this would assume that qurl is always a url and not a local file.